IDEAS.md

```diff
@@ -1,120 +1,173 @@
-# IDEAS for the future of Gupaxx
+# **IDEAS for the future of Gupaxx**
 
 
 **Theses are only ideas, everything here is still to be decided and only thoughts for now.**
-Some ideas could be done in a matter of hours, some could take months.
+- Note: Some ideas could be done in a matter of hours, some could take months.
 
-## More Decentralized
+## **Making Gupaxx more Decentralized**
 ### Synchronize source code repository on p2p network
-Github is proprietary. If Gupaxx aims to be free, it should not be only available on this platform and we should explore options to get github free.  
-We can use [Radicle](https://radicle.xyz/) to get Gupaxx on a p2p collaboration stack.  
-The code, issues and PR could be synchronized with Github.  
+- Github is proprietary. If Gupaxx aims to be free software, it should not be only available on this proprietary platform and we should explore options to be github free.  
+- We can use [Radicle](https://radicle.xyz/) to get Gupaxx on a p2p collaboration stack.  
+- We can use [Codeberg](https://codeberg.org/) as an open source alternative for Github.
+> The code, issues and PR could then also be synchronized with Github.  
+(We could create mirrors between Github, Radicle and/or Codeberg)
+
 ### Integrate a P2Pool compatible Nodes crawler
 To get rid of integrating nodes list, we can include a crawler like monero.fail inside Gupaxx.
+
```

Comment:
Space are not needed in markdown format if there is a header in between.

Reply:
https://github.com/Cyrix126/gupaxx/pull/41/commits/04d358f8a516f847f4378ff23a1a7a34d488a325